prime-router/src/testIntegration/resources/datatests/mappinginventory/cx/cx-identifier-system.hl7

```diff
@@ -1,3 +1,2 @@
-MSH|^~\&#|||||20230501102531-0400||ORU^R01^ORU_R01|12345||2.5.1|||||USA|UNICODE UTF-8
+MSH|^~\&#|||||20230501102531-0400||ORU^R01^ORU_R01|12345||2.5.1
 OBR|1|two|three|68991-9^Epidemiologically Important Information^LN^^^^^^^^^^^2.16.840.1.113883.6.1|||||||||||||||||||||unknown||||||||||||||||||||||||||||http://hl7.org/fhir/sid/us-ssn^^ISO^^PLAC
```

Comment:
Was the intention here to test the `system` mapping for OBR.53.4? It looks like OBR.53.4 is empty here, so I don't know if this is actually testing what we want it to.

Reply:
Fixed!

Reply:
Good catch